Chef/SetupWorkstation.ps1

```diff
@@ -63,5 +63,5 @@ vagrant plugin install vagrant-berkshelf
 
 # Add DevVm base machine as Vagrant box image
 Add-DevVm-Base-Machine-As-Vagrant-Box-Image# Add Vagrant box image
-vagrant box add "DevVmBaseImage" "\\kcura.corp\shares\Development\DevEx\DevVm_Install_Files\DevVmBase.box"
+vagrant box add "DevVmBaseImage" $Env:devvm_box_image_network_path

```

Comment:
Where is this environment variable setup? on our server?

Reply:
@calimetikcura Yes on the server

Reply:
Have we documented it somewhere?
If not I would just add a comment at the end of this line saying to setup this env variable for this script to work.

Reply:
Done